src/poetry/utils/env.py

```diff
@@ -255,18 +255,14 @@ def make_candidates(
                     pass
             else:
                 raise ValueError(
-                    "{} is not relative to any discovered {}sites".format(
-                        path, "writable " if writable_only else ""
-                    )
+                    f"{path} is not relative to any discovered {'writable ' if writable_only else ''}sites"
                 )
 
         results = [candidate / path for candidate in candidates if candidate]
 
         if not results and strict:
             raise RuntimeError(
-                'Unable to find a suitable destination for "{}" in {}'.format(
-                    str(path), paths_csv(self._candidates)
-                )
+                f'Unable to find a suitable destination for "{str(path)}" in {paths_csv(self._candidates)}'
```

Comment:
I don't think `str(path)` is needed, but we can use `path!s` if it is.

Reply:
removed